File: Homeworks/JSFundamentals/Fucntions/reverseNumber.js
var number = 123.45;





function reverseNumber(number) {
    var n = number.toString();
    var reverseNumber = '';
    for (var i = n.length - 1; i >= 0; i--) {
        reverseNumber += n[i];
    }
    return reverseNumber;
}

console.log(reverseNumber(number));

File: Homeworks/C#2/Methods/08.ReverseNumber/ReverseNumber.cs
using System;

//Problem 7. Reverse number

//Write a method that reverses the digits of given decimal number.

class ReverseNumber
{
    static void Main()
    {
        decimal number = decimal.Parse(Console.ReadLine());
        ReverseNumbers(number);
    }

    static void ReverseNumbers(decimal number)
    {
        string text = number.ToString();
        for (int i = text.Length - 1; i > 0; i--)
        {
            Console.Write(text[i]);
        }
    }

}

